pkgs/development/python-modules/pylama/default.nix

```diff
@@ -45,14 +47,22 @@ buildPythonPackage rec {
   ];
 
   checkInputs = [
+    # avoid infinite recursion pylint -> isort -> pylama
+    (pylint.override {
+      isort = isort.overridePythonAttrs (old: {
+        doCheck = false;
+      });
+    })
```

Comment:
The alternative to this approach is to copy the tests into a separate output and move the checks into `passthru.tests`. That way we wouldn't need to build another isort. Check the uvicorn package for example.

Reply:
But is it that costly to build them twice?

Reply:
I have no strong opinion on the matter, just wanted to throw that point in the ring.

Reply:
The disadvantage of passthru.tests is that ofborg doesn't run them for Python packages, so I prefer this.